mods/ts/rules/civilian-structures.yaml

```diff
@@ -18,6 +19,18 @@ ABAN01:
 
 ABAN02:
 	Inherits: ^CivBuilding
+	HitShape:
```

Comment:
Are you sure this is correct?

![screen shot 2017-06-06 at 22 20 10](https://user-images.githubusercontent.com/167819/26852551-8fee4ac0-4b06-11e7-8176-4541f4ce6119.png)